semgrep-core/src/engine/Eval_generic.ml

```diff
@@ -305,31 +305,32 @@ let bindings_to_env (config : Config_semgrep.t) xs =
   in
   { mvars; constant_propagation }
 
+let text_of_binding mvar mval =
```

Comment:
i usually suffix with _opt when something return an option.

Reply:
I don't know, the type contains that info already, and there's no alternate implementation to disambiguate from. I read `text_of_binding` as "that's the function to use to convert a binding to text", whereas `text_of_binding_opt` hints that there's something special about it, which isn't the case.